app/celery/tasks.py

```diff
@@ -84,3 +90,62 @@ def copy_redaction_failed_pdf(source_filename):
     # e.g. if a file has 5 days left to expire on a ONE_WEEK retention in the source bucket,
     # in the destination bucket the expiration time will be reset to 7 days left to expire
     obj.copy(copy_source, ExtraArgs={'ServerSideEncryption': 'AES256'})
+
+
+@notify_celery.task(name='create-letter-pdf')
+@statsd(namespace="template_preview")
+def create_letter_pdf(encrypted_letter_data):
```

Comment:
how about a name like `create-pdf-for-letter-template`? It makes it easier to distinguish between precompiled letter tasks and template letter tasks.

Reply:
ahh, that's a good idea, I will change that 👍